BD-2232: Transformation updates (#5691)
braze-inc/braze-docs: _docs/_user_guide/data_and_analytics/braze_currents/setting_up_currents.md

Add a partner, sometimes called a "Currents connector", by clicking the dropdown at the top of the screen.

Each partner requires a different set of configuration steps. To enable each integration, refer to our list of [available partners]({{site.baseurl}}/user_guide/data_and_analytics/braze_currents/available_partners/) and follow the instructions in their respective pages.

## Step 4: Configure events

Choose the events you wish to pass to that partner by checking from the available options. You can find listings of these events in our [Customer Behavior Events]({{site.baseurl}}/user_guide/data_and_analytics/braze_currents/event_glossary/customer_behavior_events/) and [Message Engagement Events]({{site.baseurl}}/user_guide/data_and_analytics/braze_currents/event_glossary/message_engagement_events/) libraries.

![]({% image_buster /assets/img/current4.png %})

If needed, you can learn more about our events in our [event delivery semantics]({{site.baseurl}}/user_guide/data_and_analytics/braze_currents/event_delivery_semantics/) article.

## Step 5: Field transformations

> Currents field transformations allow you to designate certain fields in Currens for removal (replace with an empty string) or hashing (applying an SHA-256 hashing algorithm). 

Reviewer: ```suggestion
Currents field transformations allow you to designate certain fields in Currents for removal (replace with an empty string) or hashing (applying an SHA-256 hashing algorithm). 
```

Add a partner, sometimes called a "Currents connector", by clicking the dropdown at the top of the screen.

Each partner requires a different set of configuration steps. To enable each integration, refer to our list of [available partners]({{site.baseurl}}/user_guide/data_and_analytics/braze_currents/available_partners/) and follow the instructions in their respective pages.

> ## Step 5: Configure events

Reviewer: ```suggestion
## Step 4: Configure events
```

Choose the events you wish to pass to that partner by checking from the available options. You can find listings of these events in our [Customer Behavior Events]({{site.baseurl}}/user_guide/data_and_analytics/braze_currents/event_glossary/customer_behavior_events/) and [Message Engagement Events]({{site.baseurl}}/user_guide/data_and_analytics/braze_currents/event_glossary/message_engagement_events/) libraries.

![]({% image_buster /assets/img/current4.png %})

If needed, you can learn more about our events in our [event delivery semantics]({{site.baseurl}}/user_guide/data_and_analytics/braze_currents/event_delivery_semantics/) article.

> ## Step 6: Field transformations

Reviewer: ```suggestion
## Step 5: Field transformations
```

Currents field transformations allow you to designate certain fields in Currens for removal (replace with an empty string) or hashing (applying an SHA-256 hashing algorithm). 

> Selecting a field for one of these transformations will apply that transformation to all events in which that field appears. 
> 
> For example, selecting `email_address` for hashing will hash the `email_address` field in Email Send, Email Open, Email Bounce, SubscriptionGroupStateChange, etc.

Reviewer: ```suggestion
Selecting a field for one of these transformations will apply that transformation to all events in which that field appears. For example, selecting `email_address` for hashing will hash the `email_address` field in Email Send, Email Open, Email Bounce, Subscription Group State Change, etc.
```

![Adding field transformations]({% image_buster /assets/img/current3.png %})

> ## Step 7: Test your integration

Reviewer: ```suggestion
## Step 6: Test your integration
```
